File: examples/example-60/js/features/support/webdriver.js
const { Before, After } = require('@cucumber/cucumber');
const { Builder } = require('selenium-webdriver');

Before({ tags: '@webdriver' }, async function () {
    this.driver = await new Builder().forBrowser('firefox')
        .usingServer('http://localhost:4444/wd/hub').build();
});

After({ tags: '@webdriver' }, async function () {
    if (this.driver) {
        await this.driver.quit();
    }
});

File: examples/example-60/ruby/features/support/webdriver.rb
require 'selenium-webdriver'

Before('@webdriver') do
    options = Selenium::WebDriver::Firefox::Options.new
    @driver = Selenium::WebDriver.for(
        :remote,
        url: 'http://localhost:4444/wd/hub',
        capabilities: options
    )
end

After('@webdriver') { @driver.quit }